src/components/scenes/TransactionDetailsScene.js

```diff
@@ -251,16 +252,22 @@ export class TransactionDetailsComponent extends React.Component<Props, State> {
     ))
   }
 
-  renderExchangeData = (symbolString: string) => {
-    const { destinationDenomination, destinationWallet, edgeTransaction, guiWallet, walletDefaultDenomProps, theme } = this.props
+
+  renderExchangeData = () => {
```

Comment:
By removing "(symbolString: string)" it breaks the code.

Reply:
Added back in.

Reply:
What is this change tested at all? You created a syntax error which could not possibly run

Reply:
Tested and rebased ontop of Develop. 
![image](https://user-images.githubusercontent.com/61520543/107074986-5ac0a380-679e-11eb-8980-161e406f8dec.png)